src/components/KCard/KCard.vue

```diff
@@ -1,244 +1,122 @@
-        <slot name="body">
-          {{ body }}
-        </slot>
-      </div>
-
-      <div
-        v-if="slots.notifications"
-        class="k-card-notifications"
-      >
-        <slot name="notifications" />
-      </div>
+    <div
+      v-if="$slots.footer"
+      class="card-footer"
+    >
+      <slot name="footer" />
     </div>
   </section>
 </template>
 
-<script lang="ts" setup>
-import { v1 as uuidv1 } from 'uuid'
-import type { BorderVariant } from '@/types'
-import { BorderVariantsArray } from '@/types'
-import type { PropType } from 'vue'
-import { computed, useSlots } from 'vue'
+<script setup lang="ts">
+import { useSlots, computed } from 'vue'
 
-const props = defineProps({
-  /**
-   * Title string if slot not used, also used for aria-label
-   */
+defineProps({
   title: {
     type: String,
     default: '',
   },
-  /**
-   * Pass body string in if slot not used
-   */
-  body: {
+  content: {
     type: String,
     default: '',
   },
   /**
-   * Set top border or no border. If neither set default will have border<br>
-   * * Options: [borderTop, noBorder]
-   */
-  borderVariant: {
-    type: String as PropType<BorderVariant>,
-    default: 'border',
-    validator: (value: BorderVariant): boolean => BorderVariantsArray.includes(value),
-  },
-  /**
-   * Sets if card has hover state<br>
-   */
-  hasHover: {
-    type: Boolean,
-    default: false,
-  },
-  hasShadow: {
-    type: Boolean,
-    default: false,
-  },
-  /**
-   * Add small status text above the card title
+   * @deprecated in favor of `content` prop
    */
-  status: {
+  body: {
     type: String,
     default: '',
-  },
-  /**
-   * Test mode - for testing only, strips out generated ids
-   */
-  testMode: {
-    type: Boolean,
-    default: false,
+    validator: (value: string) => {
+      if (value) {
+        console.warn('KCard: `body` prop has been deprecated in favor of `content` prop. See the migration guide for more details: https://alpha--kongponents.netlify.app/guide/migrating-to-version-9.html#kcard')
+      }
+
+      return true
+    },
   },
 })
 
-const titleId = computed((): string => props.testMode ? 'test-title-id-1234' : uuidv1())
-
-const contentId = computed((): string => props.testMode ? 'test-content-id-1234' : uuidv1())
-
 const slots = useSlots()
 
-const useStatusHatLayout = computed((): boolean => !!(props.status || !!slots.statusHat))
-
-const showCardHead = computed((): boolean => !!slots.actions || useStatusHatLayout.value ||
-  (!useStatusHatLayout.value && (!!props.title || !!slots.title)))
+const contentSlotName = computed((): 'body' | 'default' => { // TODO: remove this when @deprecated `body` slot is removed
+  if (slots.body && !slots.default) {
+    console.warn('KCard: `body` slot has been deprecated in favor of `default` slot. See the migration guide for more details: https://alpha--kongponents.netlify.app/guide/migrating-to-version-9.html#kcard')
```

Comment:
issue: you shouldn't log within a computed evaluation. This has to be removed

Reply:
Moved to `onMounted` hook